Makefile

```diff
@@ -9,7 +9,13 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
+vet:
+	go list ./... | grep -v "./vendor*" | xargs go vet
```

Comment:
@Random-Liu Just like next comment, we can change `grep -v "./vendor*"` to `grep -v "./vendor/*"` to minimize the impact.